src/components/ModalView/ModalView.js

```diff
@@ -0,0 +1,272 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import styles from './ModalView.module.css'
+import { Modal, Image, Row, Col, Table, Button } from 'react-bootstrap';
+import ChartTool from '../ChartTool'
+import DobleChartTool from '../DobleChartTool'
+import { setShow } from '../../redux/actions/modalViewActions'
+import { cleanSelectedPokemons } from '../../redux/actions/pokemonsActions'
+import { setShowToast } from '../../redux/actions/pokemonsActions'
+
+const ModalView = ({
+  setShow,
+  modalViewState,
+  selectedPokemons,
+  cleanSelectedPokemons,
+  setShowToast,
+  showToast
+}) => {
+  const clases = styles
+
+  const handleClick = (e) => {
```

Comment:
don't use single char variables

Reply:
handle click can be any click, it is good practice to specify what a function does, not where it is called, so it is reusable.
There are scenarios where you can call a click event function from somewhere else (very common), it will be confusing if it has this name.